src/analysis/convex/cone.lean

```diff
@@ -191,6 +191,9 @@ def map (f : E →ₗ[𝕜] F) (S : convex_cone 𝕜 E) : convex_cone 𝕜 F :=
   add_mem' := λ y₁ ⟨x₁, hx₁, hy₁⟩ y₂ ⟨x₂, hx₂, hy₂⟩, hy₁ ▸ hy₂ ▸ f.map_add x₁ x₂ ▸
     mem_image_of_mem f (S.add_mem hx₁ hx₂) }
 
+@[simp] lemma mem_map {f : E →ₗ[𝕜] F} {S : convex_cone 𝕜 E} {x : F} : x ∈ S.map f ↔ x ∈ f '' S :=
```

Comment:
This lemma looks weird to me. What does the corresponding lemma look like for `add_submonoid`?

Reply:
`y ∈ S.map f ↔ ∃ x ∈ S, f x = y`